package.json

```diff
@@ -6,11 +6,10 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && opn coverage/index.html",
     "generate-constants": "./scripts/generate-constants.js",
-    "install": "node-gyp-build \"node scripts/rebuild-core.js\"",
-    "submodule": "git submodule update --recursive --init",
+    "install": "node-gyp configure && node-gyp build",
```

Comment:
This is the same as `node-gyp rebuild`.

Reply:
I think we should be able to remove the `install` script completely. Iirc `npm` does `node-gyp rebuild` automatically for native addons.

Reply:
I kind of like explicit, it means I can look up what it does without having to know all of node.  But can do if you prefer (changed to rebuild anyway)